components/ILIAS/UI/src/Implementation/Component/Prompt/isPromptContent.php

```diff
@@ -0,0 +1,39 @@
+<?php
+
+/**
+ * This file is part of ILIAS, a powerful learning management system
+ * published by ILIAS open source e-Learning e.V.
+ *
+ * ILIAS is licensed with the GPL-3.0,
+ * see https://www.gnu.org/licenses/gpl-3.0.en.html
+ * You should have received a copy of said license along with the
+ * source code, too.
+ *
+ * If this is not the case or you just want to try ILIAS, you'll find
+ * us at:
+ * https://www.ilias.de
+ * https://github.com/ILIAS-eLearning
+ *
+ *********************************************************************/
+
+declare(strict_types=1);
+
+namespace ILIAS\UI\Implementation\Component\Prompt;
+
+use ILIAS\UI\Component\Component;
+use ILIAS\UI\Component\Button\Button;
+use ILIAS\UI\Component\Prompt\isPromptContent as IPromptContent;
+
+/**
+ * Flags some Component to be a valid content of a Prompt
+ */
+interface isPromptContent extends IPromptContent
```

Comment:
```suggestion
interface IsPromptContentInternal extends IPromptContent
```
(Consistent with e.g. `FormInputInternal`)